src/containers/LearningResourcePage/components/ActiveTopicConnections.jsx

```diff
@@ -41,27 +42,54 @@ const ActiveTopicConnections = ({
         );
       }
       return (
-        <Connections key={topic.id}>
-          <PrimaryConnectionButton
-            primary={topic.primary}
-            type="button"
-            onClick={() => setPrimaryConnection(topic.id)}>
-            {t('form.topics.primaryTopic')}
-          </PrimaryConnectionButton>
-          <BreadCrumb>
-            {breadCrumbs.map(path => (
-              <Fragment key={`${topic.id}${path.id}`}>
-                <span>{path.name}</span>
-                <ChevronRight />
-              </Fragment>
-            ))}
-          </BreadCrumb>
-          <RemoveConnectionButton
-            type="button"
-            onClick={() => removeConnection(topic.id)}>
-            <Cross />
-          </RemoveConnectionButton>
-        </Connections>
+        <Fragment key={topic.id}>
+          <Connections>
+            <PrimaryConnectionButton
+              primary={topic.primary}
+              type="button"
+              onClick={() => setPrimaryConnection(topic.id)}>
+              {t('form.topics.primaryTopic')}
+            </PrimaryConnectionButton>
+            <BreadCrumb>
+              {breadCrumbs.map(path => (
+                <Fragment key={`${topic.id}${path.id}`}>
+                  <span>{path.name}</span>
+                  <ChevronRight />
+                </Fragment>
+              ))}
+            </BreadCrumb>
+            <RemoveConnectionButton
+              type="button"
+              onClick={() => removeConnection(topic.id)}>
+              <Cross />
+            </RemoveConnectionButton>
+          </Connections>
+          {topic.topicConnections.length > 0 &&
```

Comment:
Syntes kanskje dette er i overkant mye som skjer i en render funksjon som gjør at jeg sliter med å se hele sammenhengen. Mulig å splitte opp for å øke lesbarhet?

Reply:
Ok nå?